internal/appsec/appsec.go

```diff
@@ -177,3 +179,16 @@ func (a *appsec) stop() {
 
 	a.limiter.Stop()
 }
+
+func init() {
+	appsecLog.SetBackend(appsecLog.Backend{
+		Debug: log.Debug,
+		Info:  log.Info,
+		Warn:  log.Warn,
+		Errorf: func(s string, a ...any) error {
+			err := fmt.Sprintf(s, a...)
+			log.Error(err)
+			return errors.New(err)
```

Comment:
I propose this version instead so we properly support `return log.Errorf("extra context message: %w", err)`:
```suggestion
			err := fmt.Errorf(s, a...)
			log.Error(err.Error())
			return err
```

cc @RomainMuller PTAL

Reply:
Good point: b45fe461f5cc9e4